atlas/packages/app/src/components/Layout.tsx

```diff
@@ -3,18 +3,21 @@ import { css, Global } from "@emotion/core";
 import emotionNormalize from "emotion-normalize";
 import { theme } from "@joystream/components";
 
-type LayoutProps = { children: React.ReactNode };
-
 const globalStyles = css`
 	${emotionNormalize};
 
 	body {
-		box-sizing: border-box;
 		font-family: ${theme.typography.fonts.base};
 		background: ${theme.colors.black};
 		color: ${theme.colors.gray[500]};
 	}
 
+	*,
+	*::after,
+	*::before {
+		box-sizing: border-box;
```

Comment:
Just noticed this change breaks at least one existing component:

<img width="222" alt="Screenshot 2020-07-03 at 7 37 29 PM" src="https://user-images.githubusercontent.com/12646744/86489629-3793c880-bd65-11ea-8b6d-b7dae119e3ab.png">

This is how it should look like:
<img width="223" alt="Screenshot 2020-07-03 at 7 45 19 PM" src="https://user-images.githubusercontent.com/12646744/86489802-c0aaff80-bd65-11ea-98da-ef85e0f92fa4.png">

However, I'd say we should keep this change and fix the component instead. `box-sizing: border-box` in general gives a much more predictable behavior and should be a default. That's also what all the CSS framework I've used do. @Gamaranto ping

Reply:
Yes definately, it is an easy fix anyway, we just need to adjust the margins back.